src/Microsoft.AspNetCore.DataProtection.EntityFrameworkCore/Key.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.AspNetCore.DataProtection.EntityFrameworkCore
+{
+    /// <summary>
+    /// Code first model used by <see cref="EntityFrameworkCoreXmlRepository"/>.
+    /// </summary>
+    public class Key
```

Comment:
As this is never going to be typed in full by the user anywhere a more specific name would be better.

Reply:
DataProtectionKey? Should the DbContext class be renamed to match as well, DataProtectionKeyStore?